claripy/strings.py

```diff
@@ -120,37 +132,44 @@ def StrIndexOf(input_string, substring, startIndex, bitlength):
     except ValueError:
         return BVV(-1, bitlength)
 
+
 def StrToInt(input_string, bitlength):
     """
-    Return True if the concrete value of the input_string ends with suffix
-    otherwise false.
+    Return a concrete BitVector representation of the input_string of bitlength
+    size. If it cannot be transformed into an integer its value is -1.
 
     :param input_string: the string we want to transform in an integer
-    :param bitlength: bitlength of the bitvector representing the index of the substring
+    :param bitlength: bitlength of the bitvector representing the index of the
+                      substring
 
-    :return BVV: bit-vector representation of the integer resulting from ythe string or -1 in bitvector representation
-                 if the string cannot be transformed into an integer
+    :return BVV: bitvector representation of the integer resulting from the
+                 string or -1 in bitvector representation if the string cannot
+                 be transformed into an integer
     """
     try:
         return BVV(int(input_string.value), bitlength)
     except ValueError:
         return BVV(-1, bitlength)
 
+
 def StrIsDigit(input_string):
     """
-    Returns True if the concrete value of the input_string is a digit otherwise returns False
+    Returns True if the concrete value of the input_string is a digit otherwise
+    returns False
```

Comment:
This is basically a verbose copy of the `:return:` field. Plus, it's wrong. Try "Determine whether the given string is entirely numeric"